packages/payment_qr_codes/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
```

Comment:
changelog will be generated automatically in the CI after merge (based on the commit message/PR title)